Gems/Terrain/Assets/Shaders/Terrain/TerrainPBR_ForwardPass.azsl

```diff
@@ -210,6 +212,23 @@ ForwardPassOutput TerrainPBR_MainPassPS(VSOutput input)
     ForwardPassOutput output;
 
     output.m_diffuseColor = lightingOutput.m_diffuseColor;
+
+    if (o_debugTerrainLodLevels)
+    {
+        float3 debugColor = float3(0.0, 0.0, 0.0);
+        uint lodLevel = ObjectSrg::m_patchData.m_lodLevel % 6;
+        switch (lodLevel)
+        {
+            case 0: debugColor = float3(0.1, 0.0, 0.0); break;
+            case 1: debugColor = float3(0.0, 0.1, 0.0); break;
+            case 2: debugColor = float3(0.0, 0.0, 0.1); break;
+            case 3: debugColor = float3(0.1, 0.25, 0.0); break;
+            case 4: debugColor = float3(0.25, 0.0, 0.25); break;
+            case 5: debugColor = float3(0.0, 0.25, 0.25); break;
+        }
```

Comment:
The switch from 0.1 to 0.25 totally looks like I got distracted by a baby crying or something ;p will fix. Also happy to switch to an array - it compiles down equivalently, and takes just as many lines to write, but the lines are slightly shorter so that's nice.